runtime/v1/shim/service.go

```diff
@@ -521,10 +521,7 @@ func (s *Service) checkProcesses(e runc.Exit) {
 		}
 
 		if ip, ok := p.(*process.Init); ok {
-			shouldKillAll, err := shouldKillAllOnExit(s.bundle)
-			if err != nil {
-				log.G(s.context).WithError(err).Error("failed to check shouldKillAll")
-			}
+			shouldKillAll := shouldKillAllOnExit(s.context, s.bundle)
```

Comment:
sorry for late reply. the change looks good basically. but we have other two places need to update. could you please change the `runtime/v2`? thanks

Reply:
@fuweid 
I have updated the other two files.
Please take a look.